### PR TITLE
foreach::getDoParWorkers now returns n_jobs.

### PR DESCRIPTION
**Pull request by kaneplusplus**
Fixes issue #177.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: clustermq
 Title: Evaluate Function Calls on HPC Schedulers (LSF, SGE, SLURM, PBS/Torque)
-Version: 0.8.8
+Version: 0.8.9
```
Comment (mschubert) [inline DESCRIPTION]: Please don't change version numbers with minor fixes

File: R/foreach.r [hunk 2]
```diff
@@ -3,13 +3,19 @@
 #' @param ...  List of arguments passed to the `Q` function, e.g. n_jobs
 #' @export
 register_dopar_cmq = function(...) {
+    workers <- NA
```
Comment (mschubert) [inline R/foreach.r]: The rest of the package uses `=` for assignment, this should be consistent

File: R/foreach.r [hunk 3]
```diff
@@ -3,13 +3,19 @@
...
+    dots <- list(...)
+    if ("n_jobs" %in% names(dots)) {
```
Comment (mschubert) [inline R/foreach.r]: style: no curly braces if only one expression

File: R/foreach.r [hunk 4]
```diff
@@ -3,13 +3,19 @@
...
     info = function(data, item) {
         switch(item,
                name = "clustermq",
                version = utils::packageVersion("clustermq"),
-               workers = NA)
+               workers = workers)
     }
-    foreach::setDoPar(cmq_foreach, data=list(...), info=info)
+    foreach::setDoPar(cmq_foreach, data=dots, info=info)
+
```
Comment (mschubert) [inline R/foreach.r]: style: no empty line

File: tests/testthat/test-7-foreach.r [hunk 5]
```diff
@@ -3,8 +3,14 @@ context("foreach")
 foreach = foreach::foreach
 `%dopar%` = foreach::`%dopar%`
 `%do%` = foreach::`%do%`
+getDoParWorkers = foreach::getDoParWorkers
```
Comment (mschubert) [inline tests/testthat/test-7-foreach.r]: style: this is only used once, so assigning a variable is superfluous

**Comment by kaneplusplus** [comment]
Suggested changes have been made.

**Pull request closed without merging**